GetF!t/Extensions/View+buttonpress.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+extension View {
+    func buttonPress(_ isPressed: Bool) -> some View {
+        self
+            .background(
+                !isPressed ? AnyView(normal()) : AnyView(pressed())
+            )
+    }
```

Comment:
toto je zlé. Snaž sa vôbec nepoužívať AnyView. Môžme si ukázať ako to spraviť lepšie keď prídeš do práce 🙂